rust/datafusion/tests/sql.rs

```diff
@@ -432,6 +432,40 @@ fn csv_query_count_one() {
     assert_eq!(expected, actual);
 }
 
+#[test]
+fn csv_explain() {
+    let mut ctx = ExecutionContext::new();
+    register_aggregate_csv_by_sql(&mut ctx);
+    let sql = "EXPLAIN SELECT c1 FROM aggregate_test_100 where c2 > 10";
+    let actual = execute(&mut ctx, sql).join("\n");
+    let expected = "\"logical_plan\"\t\"Projection: #c1\\n  Selection: #c2 Gt Int64(10)\\n    TableScan: aggregate_test_100 projection=None\"".to_string();
+    assert_eq!(expected, actual);
+
+    // Also, expect same result with case explain
```

Comment:
`with lowercase explain`?

Reply:
Yes, good catch. Will fix

Reply:
fixed in https://github.com/apache/arrow/pull/7959/commits/639db5f109c5861c7fb646152a48b156afde4ac3